Wordle.Api/Wordle.Api/Identity/IdentitySeed.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Microsoft.AspNetCore.Identity;
+using System.Data;
+using System.Security.Claims;
+using System;
+using Wordle.Api.Models;
+
+namespace Wordle.Api.Identity;
+public static class IdentitySeed
+{
+    public static async Task SeedAsync(UserManager<AppUser> userManager, RoleManager<IdentityRole> roleManager, WordleDbContext db)
+    {
+        // Seed Roles
+        await SeedRolesAsync(roleManager);
+
+        // Seed Admin User
+        await SeedAdminUserAsync(userManager);
+    }
+
+    private static async Task SeedRolesAsync(RoleManager<IdentityRole> roleManager)
+    {
+        // Seed Roles
+        if (!await roleManager.RoleExistsAsync(Roles.Admin))
+        {
+            await roleManager.CreateAsync(new IdentityRole(Roles.Admin));
+        }
+
+        if(!await roleManager.RoleExistsAsync(Roles.Awesome))
+        {
+            await roleManager.CreateAsync(new IdentityRole(Roles.Awesome));
+        }
+    }
+
+    private static async Task SeedAdminUserAsync(UserManager<AppUser> userManager)
+    {
+        // Seed Admin User -> Can Add/Delete Words
+        if (await userManager.FindByEmailAsync("Admin@intellitect.com") == null)
+        {
+            AppUser user = new AppUser
+            {
+                UserName = "Admin@intellitect.com",
+                Email = "Admin@intellitect.com",
+                BirthDate = new DateOnly(1999, 01, 01),
+                IsMasterOfTheUniverse = true
+            };
+
+            IdentityResult result = userManager.CreateAsync(user, "P@ssw0rd123").Result;
+
+            if (result.Succeeded)
+            {
+                await userManager.AddToRoleAsync(user, Roles.Admin);
```

Comment:
Rather than set a MasterOfTheUniverse bool to true in the new AppUser section above, you can directly add the MasterOfTheUniverse claim right here in a similar way to setting the role, such as:
await userManager.AddClaimAsync(user, new Claim(Claims.MasterOfTheUniverse, "true"));

Reply:
Got it. Thanks :)